ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/e2e/casework-regression/CaseActions/financial-plan-add-to-case.cy.ts

```diff
@@ -159,14 +159,14 @@ describe("User can add Financial Plan case action to an existing case", () => {
         Logger.Log("Ensure user must select a reason for closing the financial plan");
         closeFinancialPlanPage
             .close()
-            .hasValidationError("Please select a reason for closing the Financial Plan");
+            .hasValidationError("Reason for closure: Please enter a value");
 
         Logger.Log("Ensure notes cannot exceed 2000 characters");
         closeFinancialPlanPage
             .withReasonForClosure("Abandoned")
             .withNotesExceedingLimit()
```

Comment:
The cypress test breaks here, I had the same problem on srma, basically the new lines push the text box out of the viewing area and it can't clear it down

I think the fix is to use less new lines in our test, so it doesn't expand the text box too far

That issue will be resolved by not using javascript validation anyway, see me when you get back and I'll try and think of a way to resolve it